tests/static/js/formatters-internal/image.js

```diff
@@ -1,106 +1,197 @@
 import Formatters from 'static/js/formatters.js';
 
 describe('image formatter', () => {
-  const usUrl = 'https://a.mktgcdn.com/p/1024x768.jpg';
-  const euUrl = 'https://dyn.eu.mktgcdn.com/f/0/FOO.jpg';
-  const usImg = {url: usUrl};
-  const euImg = {
-    url: euUrl,
-    width: 1024,
-    height: 768,
-    sourceUrl: 'https://a.mktgcdn.com/p/FOO/1024x768.jpg',
+  const photoUrl = 'https://dynm.mktgcdn.com/p/FOO/2000x1000.jpg/';
+  const photoWithPaddingUrl = 'https://dynm.mktgcdn.com/p/FOO/1.0000/2000x1000.jpg/';
+  const oldFileUrl = 'https://a.mktgcdn.com/f/0/FOO.jpg';
+  const newFileUrl = 'https://a.mktgcdn.com/f/FOO.jpg';
+  const euFileUrl = 'https://a.eu.mktgcdn.com/f/0/FOO.jpg';
+
+  const photoImg = {
+    url: photoUrl,
+    width: 2000,
+    height: 1000,
```

Comment:
nit: can we pull the width and height properties out into a separate object and spread them for each type of img so they're not repeated?

Reply:
Done